metric_info.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
+
+metric_default = [
```

Comment:
Are all of these actually gauges? If any of them should be treated as counters, you should handle that case. Posting something as a gauge to SignalFx will disable the Rate/sec rollup type which might be useful if a metric only ever increases.